hs_labels/tests/test_resource_labelling.py

```diff
@@ -110,15 +112,6 @@ def test01labels(self):
         self.assertTrue(match_lists(cat.ulabels.get_resources_with_label('penalty clause'), [scratching]))
         self.assertTrue(match_lists(cat.ulabels.user_labels, ['penalty clause']))
 
-    def test02folders(self):
-        cat = self.cat  # user
-        scratching = self.scratching  # resource
-        cat.ulabels.file_resource(scratching, "penalty clause")
-        self.assertTrue(match_lists(cat.ulabels.filed_resources, [scratching]))
-        self.assertTrue(match_lists(cat.ulabels.resources_of_interest, [scratching]))
-        self.assertTrue(match_lists(scratching.rlabels.get_folder(cat), 'penalty clause'))
-        self.assertTrue(match_lists(scratching.rlabels.get_users(), [cat]))
-
```

Comment:
@pkdash after these test functions are removed, not sure whether we need to change the following test functions to be sequential, e.g., change test03favorite() to test02favorite(), etc., just to be clearer, but these are very trivial things and don't have to be done at all. It is up to you.